card/mod/machines/set/type/css.rb

```diff
@@ -55,6 +55,12 @@ def default_nest_view
   view :content_changes, mod: Abstract::Script::HtmlFormat
 end
 
+format :css do
+  view :import do
+    %{\n@import url("#{_render_url}");\n}
+  end
+end
```

Comment:
I'm surprised that (a) this is needed, and (b) it doesn't have the voo.items handling.  Is there another `import` view somewhere?